Dockerfile

```diff
@@ -7,24 +19,59 @@ COPY . /go/src/github.com/stefanprodan/mgob
 WORKDIR /go/src/github.com/stefanprodan/mgob
 
 RUN CGO_ENABLED=0 GOOS=linux \
-      go build \
-        -ldflags "-X main.version=$VERSION" \
-        -a -installsuffix cgo \
-        -o mgob github.com/stefanprodan/mgob/cmd/mgob
+    go build \
+    -ldflags "-X main.version=$VERSION" \
+    -a -installsuffix cgo \
+    -o mgob github.com/stefanprodan/mgob/cmd/mgob
 
-FROM ubuntu:focal
+FROM golang:1.15-alpine3.14 as tools-builder
+
+ARG MONGODB_TOOLS_VERSION
+
+RUN apk add --no-cache git build-base krb5-dev \
+    && git clone https://github.com/mongodb/mongo-tools.git --depth 1 --branch $MONGODB_TOOLS_VERSION
+
+WORKDIR mongo-tools
+RUN ./make build
+
+FROM alpine:3.14
```

Comment:
Latest Alpine is 3.15, please use that, less CVEs.

Reply:
👍🏼